app/controllers/api/v1/description_change_requests_controller.rb

```diff
@@ -6,8 +6,14 @@ def update
     @description_change_request = @planning_application.description_change_requests.where(id: params[:id]).first
 
     if @description_change_request.update(description_change_params)
+      previous_description = @planning_application.description
       @description_change_request.update!(state: "closed")
-      @planning_application.update!(description: @description_change_request.proposed_description) if @description_change_request.approved?
+
+      if @description_change_request.approved?
+        @description_change_request.update!(previous_description: previous_description)
```

Comment:
Rather than filling in `previous_decision` on approval, how about we fill it in on creation of the change request. eg:

```ruby
class DescriptionChangeRequest < ApplicationRecord
  before_create :set_previous_decision
  private
  def set_previous_decision
    self.previous_description = planning_application.previous_decription
  end
```

that way the value is never null, and we can just display that on the front end instead of putting logic there to decide which one we want?